internalPackageUpdate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "packageRules": [
+    {
+      "matchPackagePatterns": [
+        "^@smg-automotive\/.*"
+      ],
+      "labels": [
+        "internal"
```

Comment:
Do labels need to be created on repos or will they be added automatically?

Reply:
I am not able to test it before I merge the config. I believe it is added automatically though